### PR TITLE
[12.0][MIG] account_invoice_line_report

### PR DESCRIPTION
**Pull request by kittiu**
Done migrated

### CODE REVIEW

**Comment by kittiu** [comment]
Ready for review.
Note: I move this report to under Invoicing > Reporting > Management menu, as the old "Business Intelligence" menu is missing.

File: account_invoice_line_report/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+# Copyright 2018 Vicent Cubells - Tecnativa <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Invoice Line Report',
+    'summary': 'New view to manage invoice lines information',
+    'version': '12.0.1.0.0',
+    'category': 'Account',
+    'website': 'http://www.tecnativa.com',
```
Comment (tbaden) [inline account_invoice_line_report/__manifest__.py]: website should be https://github.com/OCA/account-invoice-reporting

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work 👍 
I did a code review: just a minor change

**Review by luismontalba — APPROVED** [review]

**Comment by kittiu** [comment]
Can this one be merged?

**Comment by rafaelbn** [comment]
Maybe @carlosdauden or @cubells could review this PR

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test

**Pull request merged**